pandas/tests/tseries/conftest.py

```diff
@@ -1,10 +1,4 @@
 import pytest
-import pandas.tseries.offsets as offsets
-
-
-@pytest.fixture(params=[getattr(offsets, o) for o in offsets.__all__])
```

Comment:
this serves the entire sub-directory, why are you removing?

Reply:
Because it isn't moved outside test_offsets and causes a "where the heck is this defined" reaction to a new reader.  Will revert.